pyproject.toml

```diff
@@ -10,6 +10,7 @@ termcolor = "^1.1.0"
 trezor = "^0.13.0"
 websockets = "^10.1"
 psutil = "^5.8.0"
+Pillow = "^9.1.0"
```

Comment:
Does it need to be capitalized?

Reply:
I think it is a capital by default. And it is the same in `firmware` - https://github.com/trezor/trezor-firmware/blob/master/pyproject.toml#L57